bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdCProjectDescriptionListener.java

```diff
@@ -18,7 +18,7 @@
 /**
  * Vendors may implement this interface as OSGi service
```

Comment:
I would reword a bit:
"Vendors may implement this interface as OSGi service
 * with a service.ranking property > 0 to to replace the default implementation {@code ClangdConfigurationFileManager}"